backend/tracim_backend/migration/versions/f889c2b59759_remove_page_slug_from_content_revision_.py

```diff
@@ -0,0 +1,46 @@
+"""remove page slug from content_revision type
+
+Revision ID: f889c2b59759
+Revises: 354d62d490ad
+Create Date: 2019-01-22 16:34:22.989215
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'f889c2b59759'
+down_revision = '354d62d490ad'
+
+
+OLD_SLUG = 'page'
+NEW_SLUG = 'html-document'
+NEW_SLUG_FILE_EXTENSION = '.document.html'
+
+content_revisions = sa.Table(
+    'content_revisions',
+    sa.MetaData(),
+    sa.Column('type'),
+    sa.Column('file_extension'),
+)
+
+
+def upgrade():
+    connection = op.get_bind()
+    connection.execute(
+        content_revisions.update().where(
+            content_revisions.c.type == OLD_SLUG
+        ).values(
+            type=NEW_SLUG,
+            file_extension=NEW_SLUG_FILE_EXTENSION
+        )
+    )
+
+def downgrade():
+    """ Downgrade is not able to retrieve content with old slug and content
+    with new as both are valid before this. Stay with NEW_SLUG is better, so
+    don't do anything
+    """
+    pass
```

Comment:
Do you realize that you break downgrade-capacity of tracim, here? You can't decide as a developer to break this, this kind of decision must be discussed.

Reply:
In fact, downgrading problem is not that new. New/Updated html-document are not supported for downgrading.
 Note:  Downgrade issue happened only with tracim code older than https://github.com/tracim/tracim/commit/a05c27a7869b311755590359b2ae587f9df84d64 as old not updated page can't be read anymore in old version of tracim.

We can converted all content to page in this migration to avoid issue as downgrading. This may work but that's not really correct. In fact, migration tree lack a "downgrading only" migration to revert all html-document to page.

Reply:
I did not see any documentation or warning about impossibility to downgrade.

> We can converted all html-document to page in this migration to avoid issue as downgrading. This may work but that's not really correct. In fact, migration tree lack a "downgrading only" migration to revert all html-document to page at the correct level.

This is your work to identify this kind of issue when you implement features / refactoring. You should have added a migration, or at least document that downgrade is not possible in this specific case

Reply:
So, please add a documentation information which explain that downgrade from tracim v2 to tracim v1 (if it's this case) is not supported

Reply:
then the fix will be ok (to be tester by @PhilippeAccorsi with our production dump)